src/app/app.component.ts

```diff
@@ -1,5 +1,16 @@
 import { Component } from '@angular/core';
 import { buttonView, buttonType, iconPosition } from './core/models/button.enum';
+import { counterStyle } from 'src/app/core/models/counter.enum';
```

Comment:
must be removed. Use git stash to use testing code comfortable

https://git-scm.com/docs/git-stash